app/scripts/api/snapshot/Transporter.js

```
@@ -697,13 +750,16 @@ define(function(require) {
 
         // handler for postMessages received from the viewer
         var messageEventHandler = function(event) {
+            var message;
             try {
-                var message = JSON.parse(event.data);
-                self.capiMessageHandler(message);
+                message = JSON.parse(event.data);
             } catch (e) {
                 window.console.log(e);
             }
 
+            if (message) {
```

Resolved - my confusion, when errors in message recieving callbacks are not caught we want to leave them uncaught and fail loudly!